### PR TITLE
sdk/cameras/camera_itof.cpp: print chip/imager errors

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: sdk/include/aditof/adsd_errs.h [hunk 1]
```diff
@@ -46,7 +46,6 @@ namespace aditof {
  * Included with aditof/camera.h
  * For example: 
  *    ADSDErrors err;
- *    LOG(INFO) << ADSDErrors::ADSD3500_STATUS_INVALID_MODE;
```
Comment (dNechita) [inline sdk/include/aditof/adsd_errs.h:49]: Why the need to remove this?

File: sdk/include/aditof/adsd_errs.h [hunk 2]
```diff
@@ -169,204 +168,202 @@ class ADSDErrors {
...
+                /**
      * @brief PLLLOCK error location 3.
      * Imager error codes read via "Get Imager Error Code" (0x0038) command
      * Note, this only valid if "Get Status" (0x0020) reports a value of "ADSD3500_STATUS_IMAGER_ERROR" (0x0010)
      */
-    static const uint16_t ADSD3100_ERR_PLLLOCK_LOCK3 =
+    const uint16_t ADSD3100_ERR_PLLLOCK_LOCK3 =
         0x000C; // PLLLOCK error location 3.
-    /**
+                /**
```
Comment (dNechita) [inline sdk/include/aditof/adsd_errs.h:283]: this looks weird (not aligned properly)

File: sdk/include/aditof/adsd_errs.h (hunk 1 above)
Comment (SeptimiuVana) [reply sdk/include/aditof/adsd_errs.h:49]: Those variables are no longer static, so we can't print them like this.

File: sdk/include/aditof/adsd_errs.h (hunk 2 above)
Comment (SeptimiuVana) [reply sdk/include/aditof/adsd_errs.h]: Fixed.

**Review by dNechita — APPROVED** [review]

**Pull request merged**